core/src/test/java/com/graphhopper/routing/RandomizedRoutingTest.java

```diff
@@ -132,15 +133,15 @@ public void init() {
         // fail sometimes for edge-based algorithms, #1631, but maybe we can should disable different fwd/bwd speeds
         // only for loops instead?
         encoder = new CarFlagEncoder(5, 5, maxTurnCosts);
-        encodingManager = EncodingManager.create(encoder);
+        encodingManager = new EncodingManager.Builder().add(encoder).add(Subnetwork.create("car")).build();
         graph = new GraphBuilder(encodingManager)
                 .setCHConfigStrings("p1|car|fastest|node", "p2|car|fastest|edge")
                 .setDir(dir)
                 .create();
         turnCostStorage = graph.getTurnCostStorage();
         chConfigs = graph.getCHConfigs();
         // important: for LM preparation we need to use a weighting without turn costs #1960
-        lmConfig = new LMConfig("config", chConfigs.get(0).getWeighting());
+        lmConfig = new LMConfig("car", chConfigs.get(0).getWeighting());
         weighting = traversalMode.isEdgeBased() ? chConfigs.get(1).getWeighting() : chConfigs.get(0).getWeighting();
```

Comment:
@easbar do you think it is fine when we force to add a subnetwork EncodedValue? See also below for the other tests that needed a change due to the introduced check in LandmarkStorage.

Reply:
Ok it seems for now this is a good choice. And if this makes problems we can still change it later (use separate edge-based TarjanSCC for LM or so).